### DISCUSSION TITLE
How to make label with 2 lines for title column in table ?

### DISCUSSION THREAD

**Discussion by jigujigu** [filamentphp/filament#6286] [Help] [answered]
I want to make a label become  2 lines:

Home
Number

instead  just 1 line:
<img width="143" alt="image" src="https://user-images.githubusercontent.com/6214964/233769802-e9c01ada-73f5-4fda-8360-fb3ff3ab5437.png">

Below is code, which I already tried but failed : 
```
Tables\Columns\TextColumn::make('home_number')->label('Home' . chr(10) . 'Number'),

Tables\Columns\TextColumn::make('home_number')->label('Home' . .PHP_EOL  . 'Number'),

Tables\Columns\TextColumn::make('home_number')->label('Home  \r\n Number'),
```

Thank you

**Comment by jigujigu** [comment]
anyone can help me please.....

Thank you

**Comment by ArtMin96** [comment, marked as answer]
Option 1:
```php
->label(new HtmlString('Home <br /> number'))

// OR

->label(new HtmlString(nl2br("Home \n number")))
```

**Reply by jigujigu** [reply]
note : don't forget use Illuminate\Support\HtmlString;
thank you...
